docker/pypi/wmagent/install.sh

```diff
@@ -0,0 +1,179 @@
+
+echo "Done $stepMsg!" && echo
+echo "-----------------------------------------------------------------------"
+
+stepMsg="Generating and preserving current build id"
+echo "-----------------------------------------------------------------------"
+echo "Start $stepMsg"
+echo $RANDOM |sha256sum |awk '{print $1}' > $WMA_ROOT_DIR/.dockerBuildId
+echo "WMA_BUILD_ID:`cat $WMA_ROOT_DIR/.dockerBuildId`"
+echo "WMA_BUILD_ID preserved at: $WMA_ROOT_DIR/.dockerBuildId "
+echo "Done $stepMsg!" && echo
+echo "-----------------------------------------------------------------------"
+
+stepMsg="Replace the current /data/manage script coming from 'dmwm-base' image with a symlink link"
+echo "-----------------------------------------------------------------------"
+echo "Start $stepMsg"
+[[ -f /data/manage ]] && rm -f /data/manage && ln -s $WMA_MANAGE_DIR/manage /data/manage
+echo "Done $stepMsg!" && echo
+echo "-----------------------------------------------------------------------"
+
+stepMsg="Creating all runtime aliases for the WMA_USER"
+echo "-----------------------------------------------------------------------"
+echo "Start $stepMsg"
+set -x
+cat <<EOF >> /home/${WMA_USER}/.bashrc
+
+alias lll="ls -lathr"
+alias ls="ls --color=auto"
+alias ll='ls -la --color=auto'
+
+alias condorq='condor_q -format "%i." ClusterID -format "%s " ProcId -format " %i " JobStatus  -format " %d " ServerTime-EnteredCurrentStatus -format "%s" UserLog -format " %s\n" DESIRED_Sites'
+alias condorqrunning='condor_q -constraint JobStatus==2 -format "%i." ClusterID -format "%s " ProcId -format " %i " JobStatus  -format " %d " ServerTime-EnteredCurrentStatus -format "%s" UserLog -format " %s\n" DESIRED_Sites'
+alias agentenv='source $WMA_ENV_FILE'
+agentenv
+alias manage=\$manage
+
+# Aliases for Tier0-Ops.
+alias runningagent="ps aux | egrep 'couch|wmcore|mysql|beam'"
+alias foldersize="du -h --max-depth=1 | sort -hr"
+
+# Better curl command
+alias scurl='curl -k --cert ${CERT_DIR}/servicecert.pem --key ${CERT_DIR}/servicekey.pem'
+
+# set WMAgent docker specific bash prompt:
+export PS1="(WMAgent-\$WMA_TAG) [\u@\h:\W]\$ "
+export WMA_BUILD_ID=\$(cat $WMA_ROOT_DIR/.dockerBuildId)
+export WMAGENTPY3_ROOT=\$WMA_INSTALL_DIR/wmagent
+export WMAGENTPY3_VERSION=\$WMA_TAG
+export PATH=\$WMA_INSTALL_DIR/wmagent/bin\${PATH:+:\$PATH}
+EOF
+
+set +x
+echo "Done $stepMsg!" && echo
+echo "-----------------------------------------------------------------------"
+
+stepMsg="Populating cronjob with utilitarian scripts for the WMA_USER"
+echo "-----------------------------------------------------------------------"
+echo "Start $stepMsg"
+
+# TODO: These executable flags we should consider fixing them for all *.sh
+#       scripts under the /deploy top level area in the WMCore github repository
+chmod +x $WMA_DEPLOY_DIR/deploy/renew_proxy.sh $WMA_DEPLOY_DIR/deploy/restartComponent.sh
```

Comment:
And this is, because you have used a `wmagent` version lower than the minimal supported one. which is again: `2.2.1rc3`. Everything before that is crating a broken python package. This latest version is uploaded in pypi so you could safely use it:
https://pypi.org/project/wmagent/#history